app/index.html

```diff
@@ -28,7 +28,7 @@
 <link rel="icon" href="img/icons/512.png" sizes="512x512">
 <link rel="icon" href="img/icon.svg" sizes="any" type="image/svg+xml">
 <title>Project Link</title>
-<script defer src="js/initapp.js"></script>
+<script src="js/alameda.js" data-main="js/bootstrap"></script>
```

Comment:
> If your app uses completely independent set of JavaScript files, you can bundle them separately (for instance, a main application code base and the analytics code) and load them with defer and async. This way it doesn't matter if one is loaded better or after the other because they're completely independent and their loading order is not relevant.

Yeah, I understand, I meant what exactly do you want load separately in foxapp at the moment?